docs/ToolsAndExamples/shareModal.md

```diff
@@ -1,8 +1,10 @@
 ---
-sidebar_position: 3
+sidebar_position: 1
 ---
 
-# Share Modal
+# Tools 🛠
```

Comment:
This section should have the folder structure like this (and not have all the content on a single pa
<img width="303" alt="Screen Shot 2022-10-18 at 1 00 23 PM" src="https://user-images.githubusercontent.com/5404516/196497432-3145c265-d930-4f78-bb7a-56293d7c8e00.png">
ge):